Sources/Parsing/Parsers/Int.swift

```diff
@@ -119,6 +123,7 @@ extension Parsers {
       }
       guard length > 0
```

Comment:
I think `guard length > parsedSign ? 1 : 0` would allow to bypass the check of l126

Reply:
That's a great idea, thanks!